pkg/services/alerting/notifiers/opsgenie.go

```diff
@@ -156,10 +174,15 @@ func (on *OpsGenieNotifier) createAlert(evalContext *alerting.EvalContext) error
 		customData += fmt.Sprintf("%s: %v\n", evt.Metric, evt.Value)
 	}
 
+	aliasPrefix := "alertId"
```

Comment:
You could add a function to avoid repetition?

```
func (on *OpsGenieNotifier) getAlias(id int64) string {
    aliasPrefix := "alertId"
    if on.AliasPrefix != "" {
		aliasPrefix = on.AliasPrefix
	}
    return fmt.Sprintf("%s-%d", aliasPrefix, evalContext.Rule.ID))
}
```